style.css

```diff
@@ -168,6 +168,106 @@
     list-style-type: none;
   }
 
+  .pop-up-modal {
+    position: absolute;
+    background-color: white;
+    margin-top: 70px;
+    border-radius: 0.5rem;
+    width: 375px;
+    height: auto;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+
+  .overlay {
+    display: none;
+    position: fixed;
+    z-index: 1;
+    left: 0;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    overflow: auto;
+    background-color: rgba(0, 0, 0, 0.8);
+  }
+
+  .img-cont-pop {
+    display: block;
+    text-align: center;
+    padding-top: 10px;
+  }
+
+  #img {
+    width: 350px;
+    height: auto;
+  }
+
+  .close-btn-cont {
+    display: flex;
+    justify-content: flex-end;
+    height: 20px;
+    width: 350px;
+    margin: 200px 20px 0 0;
+  }
+
+  .space-bet {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 30px 0 30px;
+  }
+
+  .flex-btn {
+    display: none;
+  }
+
+  .flex-btn-mob {
+    display: flex;
+    justify-content: center;
+  }
+
+  .live {
+    display: none;
+  }
+
+  .live-mob {
+    background: #26815b;
+    border-radius: 4px;
+    width: 122px;
+    height: 48px;
+    font-family: "Inter", sans-serif;
+    color: white;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    margin: 20px 5px 20px 5px;
+    border: none;
+    text-decoration: none;
+  }
+
+  #name {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+  }
+
+  .buttons-tech {
+    margin: 0 0 0 25px;
+  }
+
```

Comment:
- [ ] Good job on adding the list of skills but please change the background color of them as you can see it on [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468).

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/156473544-899038d4-4dfa-45b6-bdb8-02893a3afe53.png)

Reply:
okay, Thank you, I change the background and the font color :+1: